core/.js/src/main/scala/io/udash/Application.scala

```diff
@@ -46,15 +46,18 @@ class Application[HierarchyRoot >: Null <: GState[HierarchyRoot] : PropertyCreat
   /**
    * Starts the application using selectors to find root element. Handles waiting for document to be ready.
    *
-   * @param selectors A DOMString containing one or more selectors to match.
-   *                  This string must be a valid CSS selector string; if it isn't, a native SyntaxError exception is thrown.
-   *                  See https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors.
-   * @param callback  Callback ran after the application is started.
+   * @param selectors            A DOMString containing one or more selectors to match.
+   *                             This string must be a valid CSS selector string; if it isn't, a native SyntaxError exception is thrown.
+   *                             See https://developer.mozilla.org/en-US/docs/Web/API/Document_object_model/Locating_DOM_elements_using_selectors.
+   * @param onContentLoaded      Callback ran on the application root element before the application is started.
```

Comment:
In order to precisely indicate when the callback gets executed, I'd name it `onDocumentReady` or modify its description accordingly (e.g. _before the application is started but once the DOM document is ready_).

Reply:
Did both.